lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -2596,6 +2596,24 @@ LogicalResult ConvertAtenOp<AtenViewOp>::matchAndRewrite(
     return rewriter.notifyMatchFailure(op,
                                        "size must consist of Scalar constants");
 
+  // # the size -1 is inferred from other dimensions
```

Comment:
Can you add e2e tests that test the new additions?

Reply:
I just add a view test in basic.mlir. Would it be enough?

Reply:
That's not enough. See: https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#dos-and-donts-for-unit-vs-end-to-end-testing

> DO use an [end-to-end test](https://github.com/llvm/torch-mlir/blob/main/docs/adding_an_e2e_test.md) if you are implementing a new op or extending the support for an existing op.

Also, the tests in basic.mlir are not needed.

Reply:
Ok, just delete the file in basic.mlir. And add an e2e test `ViewNegativeStaticModule_basic`

Reply:
Done